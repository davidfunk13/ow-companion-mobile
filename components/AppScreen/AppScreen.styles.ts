import { StyleSheet, } from "react-native";
import theme from "../../constants/themes";

const styles = StyleSheet.create({
	background: {
		height:          "100%",
		padding:         10,
		backgroundColor: theme.light.background,
	}, 
});

export default styles;
