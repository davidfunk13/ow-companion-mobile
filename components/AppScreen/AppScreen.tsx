import { FC, } from "hoist-non-react-statics/node_modules/@types/react";
import React from "react";
import { SafeAreaView, } from "react-native";

interface IAppScreenProps {

}

const AppScreen: FC<IAppScreenProps> = ({ children, }) => {
	return (
		<SafeAreaView >
			{children}
		</SafeAreaView>
	);
};

export default AppScreen;
