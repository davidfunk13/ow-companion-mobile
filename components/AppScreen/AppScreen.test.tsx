import React from "react";
import renderer from "react-test-renderer";

import AppScreen from "./AppScreen";

describe("<App />", () => {
	it("renders correctly", () => {
		const tree = renderer.create(<AppScreen />).toJSON();

		expect(tree).toMatchSnapshot();
	});
});
