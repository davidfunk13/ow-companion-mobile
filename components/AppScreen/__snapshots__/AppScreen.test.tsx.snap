// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> renders correctly 1`] = `
<RCTSafeAreaView
  emulateUnlessSupported={true}
  style={
    Object {
      "backgroundColor": "#ffffff",
      "height": "100%",
      "padding": 10,
    }
  }
/>
`;
